jinahub/rankers/LightGBMRanker/README.md

```diff
@@ -0,0 +1,99 @@
+# ✨ LightGBMRanker
+
+**LightGBMRanker** is a Jina ranker, using the [LightGBM](https://github.com/microsoft/LightGBM) library, mode specifically, the `LightGBMRanker` is used for learning-to-rank.
+
+`LightGBMRanker` retrieves `query_features`, `match_features` and `relevance_label` stored inside `Document` object from `DocumentArray`, and builds a feature-label dataset to train the model.
+
+**Table of Contents**
+
+- [🌱 Prerequisites](#-prerequisites)
+- [🚀 Usages](#-usages)
+- [🎉️ Example](#%EF%B8%8F-example)
+- [🔍️ Reference](#%EF%B8%8F-reference)
+
+
+## 🌱 Prerequisites
+
+- This Executor works on Python 3.7 and 3.8. 
+- Make sure to install the [requirements](requirements.txt)
```

Comment:
Should only be necessary when developing locally, not with `jinahub+docker` or `jinahub`, I believe